.jenkins/pipelines/nightly.Jenkinsfile

```diff
@@ -50,4 +86,17 @@ pipeline {
             }
         }
     }
+    post {
+        always {
+            build(
+                job: "../Standalone-Pipelines/Send-Email",
+                parameters: [
+                    string(name: "REPOSITORY", value: params.REPOSITORY),
+                    string(name: "BRANCH", value: params.BRANCH),
+                    string(name: "EMAIL_SUBJECT", value: "Jenkins: ${env.JOB_NAME} [#${env.BUILD_NUMBER}] status is ${currentBuild.currentResult}"),
```

Comment:
Any preference for a different email subject?
Maybe something like `[Jenkins] [${currentBuild.currentResult}] [${env.JOB_NAME}] [#${env.BUILD_NUMBER}]`

Reply:
Looks good to me. It'd be nice to have the same/similar format for all the emails